lms/templates/static_htmlbook.html

```diff
@@ -93,20 +101,20 @@
     $(document).ready(function() {
         var options = {};
         %if 'url' in textbook:
-            options.url = "${textbook['url']}";
+            options.url = "${textbook['url'] | n, js_escaped_string}";
         %endif
         %if 'chapters' in textbook:
             var chptrs = [];
             %for chap in textbook['chapters']:
-                chptrs.push("${chap['url']}");
+                chptrs.push("${chap['url'] | n, js_escaped_string}");
             %endfor
             options.chapters = chptrs;
         %endif
         %if chapter is not None:
-            options.chapterNum = ${chapter};
+            options.chapterNum = ${int(chapter) | n, dump_js_escaped_json};
```

Comment:
@nedbat: This is difficult to test and introduces more changes than I'd like.  How do you feel about us making a reasonable guess about when and if to skip escaping on legacy code fixes?  For example, I would use the following:
```
options.chapterNum = ${chapter | n, decode.utf8};
```
I'm asking both about this PR, and as we give guidance on future PRs that are difficult to test.